src/Expression/NegateDecorator.php

```diff
@@ -0,0 +1,57 @@
+<?php
+declare(strict_types=1);
+
+namespace Arkitect\Expression;
+
+use Arkitect\Analyzer\ClassDescription;
+use Arkitect\Rules\Violation;
+use Arkitect\Rules\ViolationMessage;
+use Arkitect\Rules\Violations;
+
+class NegateDecorator implements Expression
```

Comment:
What about naming it just `Not`? I could also simplify other expressions

Reply:
Totally agree!